src/main/java/banking/database/AccountDao.java

```diff
@@ -0,0 +1,97 @@
+package banking.database;
+
+import banking.model.Account;
+import banking.model.Card;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Optional;
+
+public class AccountDao {
+
+    public static final String SQL_GET = "SELECT * FROM account WHERE cardNumber = ?;";
+    public static final String SQL_GET_IF_CORRECT = "SELECT * FROM account WHERE cardNumber = ? AND cardPin = ?;";
```

Comment:
What means if correct?

Reply:
If correct number and PIN. Renamed that field to GET_BY_CREDENTIALS